packages/chrome/src/styled/sheet/StyledSheetHeader.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet_header';
+
+export const StyledSheetHeader = styled.header.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<ThemeProps<DefaultTheme>>`
+  display: flex;
+  flex-flow: ${props => (props.theme.rtl ? 'row-reverse' : 'row')} wrap;
+  align-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid ${props => props.theme.palette.grey[300]};
+  padding: 20px;
+  min-height: 40px;
```

Comment:
[Similar](https://github.com/zendeskgarden/react-components/pull/1231/files?authenticity_token=ocFDB1GmyeZM0qKO%2B%2B%2BaueEsdBPb4UU%2F%2FUUGDkPrbzizvqjKB%2FQzJUVHh0bjsIMD1%2FkviL9azLOv18Ws9Ws0vA%3D%3D&file-filters%5B%5D=.lock&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r741539973)

Reply:
Like the `StyledBody` it was specific to the `Sheet` design spec.